src/flatc.cpp

```diff
@@ -352,6 +383,10 @@ int main(int argc, const char *argv[]) {
         }
       }
 
+      if(!has_grpc_lang_set && grpc_enabled) {
```

Comment:
This error check can be removed entirely, along with that variable. If someone writes `--grpc` without a language, you'll get `no options: specify at least one generator.` already.